networking/vnic/ipvlan/basic/runtest.sh

```diff
@@ -201,8 +201,26 @@ rlPhaseStartTest "local_netns"
 		do
 			rlRun "ip netns exec server ethtool -K ipvlan_s ${dev_features[$feature_id]}"
 			rlRun "ip netns exec server ethtool -k ipvlan_s"
-			rlRun "ip netns exec client ping 2.2.2.171 -c 5"
-			rlRun "ip netns exec client ping6 2222::171 -c 5"
+			for i in {1..10}
+			do
+				if ip netns exec client ping 2.2.2.171 -c 1
+				then
+					break
+				else
+					sleep 1
+				fi
+			done
+			rlRun "ip netns exec client ping 2.2.2.171 -c 2"
+			for i in {1..10}
+			do
+				if ip netns exec client ping6 2222::171 -c 1
+				then
+					break
+				else
+					sleep 1
+				fi
```

Comment:
Would it be possible to do something similar for the `sleep 5` above? In any case, but especially if we are to apply this kind of solution for the `sleep 5` calls above, would it be easy to make a function out of this? Something like `waitFor` giving `10` and the check command as parameters?

Reply:
waitFor make sense, but it doesn't apply for the sleep 5 above, as I don't know how to check if tcpdump is ready of have finished capturing packets.
I would improve with waitFor and submit patchv2. and close this pull request.
thanks for your suggestion